docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -12,7 +12,7 @@ toc::[]
 [[setup-driver-plugin-overview]]
 == Overview
 {project} uses Docker Machine and its driver plug-in architecture to provide a consistent way to manage the OpenShift VM.
-{project} embeds VirtualBox and VMware Fusion drivers so no additional steps are required to use them.
+{project} embeds VirtualBox driver so no additional steps are required to use it.
```

Comment:
This feels like cleaner writing to me:

```adoc
{project} embeds a VirtualBox driver.
No additional steps are required to use it.
```

At the very least, it should be "a VirtualBox driver".

Reply:
@robin-owen a VirtualBox driver" does not sound correct to me as there is only one implementation for the Virtualbox driver. So it should be "the VirtualBox driver".

Reply:
To be nitpicking in that case, it is called a 'driver plugin' in all the documentation and references at docker/machine. This is also what we use in our configuration topic.

Reply:
> "a VirtualBox driver" does not sound correct to me as there is only one implementation for the Virtualbox driver. So it should be "the VirtualBox driver".

I wasn't sure if there was more than one implementation -- in this case, yes, "the" makes more sense.

> To be nitpicking

This isn't nitpicking if it's more technically accurate to call it "the VirtualBox driver plug-in".